src/__snapshots__/cookie-control-config.test.ts.snap

```diff
@@ -24,7 +24,12 @@ Object {
     "__ngat_1.2",
     "__nrpa_2.2",
     "IdAM",
-    "IdAM-*",
+    "IdAMC*",
```

Comment:
Don't we want IdAM-* too?

Reply:
i think i got the cookie wrong for this one and John's right. 

the cookie used by local/dev sites, which don't use redis is: IdAMC[x]

all the sites deployed to servers though just have a single idam cookie called IdAM, which just contains a session key, so this hasn't affected anything except dev machines not using redis.

Reply:
Great, thanks for clarifying, will approve then!